### DISCUSSION TITLE
Desktop app update popup could be less prominent

### DISCUSSION THREAD

**Discussion by alstr** [tutao/tutanota#3958] [Ideas]
It's great that the desktop app is updated so frequently, but the "Update for Tutanota Desktop available (x.y.z)" popups that require actioning are a bit disruptive to your workflow given the regularity of updates.

Also, if you postpone an update, the alert appears again when you close/reopen the window. I don't leave the window open on Mac, so I get this popup every time I check emails unless I update, which I don't always want to do at that exact moment.

* Could we make an icon somewhere that shows that an update is available, rather than trigger the popup?
* If postponing, could we hide the alert for 24 hours or however long, or until next boot? Or just auto update at next boot?

Appreciate that it is not necessarily that simple, so I'm just throwing a few ideas out there.

**Comment by charlag** [comment]
@alstr that's valid. Our normal release cadence is two weeks but the previous release got delayed so we put ad-hoc release inbetween. I think your proposals are valid but if it will be only once every two weeks it shouldn't be an issue.
We will consider this improvement, especially for platforms where update is applied on exit anyway (macOS and Linux AppImage)
